data/kamvas-pro-13.tablet

```diff
@@ -30,15 +30,26 @@
 #    |                       |
 #    *-----------------------*
 #
+# Kamva Pro 12 Tablet Monitor Touch Strip:
+# NOTE: It does not work also in the Pro 12
+# evtest reports the touch strip as a ABS_WHEEL with code 8 and reports ABS_MISC with code 40
+#
+#    *-----------------------*
+#    |                       |
+#  A |                       |
+#  a |        DISPLAY        |
+#    |                       |
+#    *-----------------------*
+#
 # Also matches:
 # GAOMON S56K which doesn't have buttons but since HUION re-uses usb IDs
 # there is nothing we can do about this.
 
 [Device]
-Name=Huion Kamvas Pro 13
-ModelName=GT-133
+Name=Huion Kamvas Pro 12/13
+ModelName=GT-116/GT-133
 Class=Cintiq
-DeviceMatch=usb:256c:006e:Tablet Monitor Pen;usb:256c:006e:Tablet Monitor Pad;
+DeviceMatch=usb:256c:006e:Tablet Monitor Pen;usb:256c:006e:Tablet Monitor Pad;usb:256c:006e:Tablet Monitor
```

Comment:
To clarify here: does the device *only* need the new `Tablet Monitor` match or does it require the other ones too?

If the new match alone is sufficient, it's better to have a separate data file here please.

Reply:
got confused myself reading this again, so let's try to clarify: does your device match *only* the newly introduced "tablet monitor" device, or does it also export the various `Pen`, `Pad` devices?

And if not, did you update the kernel recently, because the `Pen` etc suffix is appended by the kernel driver, so missing that means something's amiss there. There was a libinput bug recently with a similar issue but I can't find ti right now, let's put the bat signal out for @JoseExposito

Reply:
I think that you refer to this PR:
https://github.com/linuxwacom/libwacom/pull/542#issuecomment-1373428174

The same advice applies in this case. Try to uninstall DIGImend if you re using it and update your kernel. When using a recent kernel, DIGImend actually removes features rather than adding them.

Reply:
Thanks! I'm going to close this one for now and if the actual issue is different, we can re-open it.